sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/ParquetVectorUpdaterFactory.java

```diff
@@ -181,9 +181,8 @@ public ParquetVectorUpdater getUpdater(ColumnDescriptor descriptor, DataType spa
         } else if (sparkType == DataTypes.BinaryType) {
           return new FixedLenByteArrayUpdater(arrayLen);
         }
-        break;
-      default:
-        break;
+      }
+      default -> {}
```

Comment:
Just to confirm, can we move 

```java
throw constructConvertNotSupportedException(descriptor, sparkType);
``` 

from line 190 into the default branch?

Reply:
Good idea.

Reply:
Got